package.json

```diff
@@ -1,21 +1,33 @@
 {
-  "name": "sumo-ot-js",
+  "name": "sumo-opentelemetry",
   "version": "0.0.0",
-  "private": true,
+  "description": "Sumo Logic all-in-one library for OpenTelemetry tracing auto-instrumentation in web",
```

Comment:
I don't like the description:
* Does not says whether it is frontend or backend.
* Does not describe what it does
* ....

Adds OpenTelemetry tracing auto-instrumentation in the browser. Collects spans on network events and sends them to Sumo Logic.